kotsadm/pkg/handlers/restore.go

```diff
@@ -189,6 +189,36 @@ func GetRestoreStatus(w http.ResponseWriter, r *http.Request) {
 	JSON(w, 200, response)
 }
 
+func CancelRestore(w http.ResponseWriter, r *http.Request) {
+	if handleOptionsRequest(w, r) {
+		return
+	}
+
+	if err := requireValidSession(w, r); err != nil {
+		logger.Error(err)
+		return
+	}
+
+	appSlug := mux.Vars(r)["appSlug"]
+
+	foundApp, err := store.GetStore().GetAppFromSlug(appSlug)
+	if err != nil {
+		err = errors.Wrap(err, "failed to get app from app slug")
+		logger.Error(err)
+		w.WriteHeader(500)
```

Comment:
let's use the status codes from the http package
```suggestion
		w.WriteHeader(http.StatusInternalServerError)
```

Reply:
love it! will change